### DISCUSSION TITLE
Double Events

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#8251] [Unsupported Versions]
Hi,
on my instance (Manager+Receiver+Search+2 Sensors) I see all the events duplicated, the _index suggests that the web application (both kibana and soc) is querying both the manager and the search node.
Is there any way to convince them to ask only the manager?

Thanks in advance

**Comment by brazorfajeje** [comment]
If from kibana management console I remove the searchnode from "Remote Clusters" the problem does not occour again.
Is this an acceptable solution? Is this a permanent solution?

**Comment by dougburks** [comment]
Something doesn't seem right. You shouldn't be seeing double events in the default configuration. Assuming your Manager is configured as a Manager-Search and you have a separate Search Node, then they should be load balancing and only storing events once and you shouldn't need to remove the search node from the configuration. 

Did you make any custom changes prior to noticing double events?

**Comment by brazorfajeje** [comment]
The manager is not manager-search but manager only, and the logs are actually stored only on the search node.
I have enabled Elastic Clustering, but in the remote clusters I can see also the search node within the remote clusters.

**Reply by dougburks** [reply]
Did you enable Elastic clustering in our installer or did you manually enable it after installation?

**Reply by brazorfajeje** [reply]
I enabled it during the installation with the installer

**Comment by weslambert** [comment]
Redacting as necessary, what is the output of the following from the manager?

`so-elasticsearch-query _cluster/settings?pretty`
